doc/api/stream.md

```diff
@@ -580,9 +580,8 @@ The Readable can switch back to paused mode using one of the following:
 
 * If there are no pipe destinations, by calling the
   [`stream.pause()`][stream-pause] method.
-* If there are pipe destinations, by removing any [`'data'`][] event
-  handlers, and removing all pipe destinations by calling the
-  [`stream.unpipe()`][] method.
+* If there are pipe destinations, by removing all pipe destinations by
+  calling the [`stream.unpipe()`][] method.
```

Comment:
only calling `unpipe()` will make stream to `paused` mode. see the following example.

```
const { Readable, Writable } = require('stream');

const r = new Readable();
const w = new Writable();

r._read = () => {};
w._write = () => {};

r.pipe(w);
r.on('data', () => {});

r.unpipe();
console.log(r._readableState.flowing); // the output is false
```

Reply:
quick grammar nit on this: `by removing all pipe destinations by ...` the sentence should be something like:

```
Multiple pipe destinations may be removed by calling the [`stream.unpipe()`][] method.
```

Reply:
fixed.